recipes/boost/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "1.60.0" %}
+
+package:
+  name: boost
+  version: {{ version }}
+
+source:
+  fn:  boost_{{ version.replace(".", "_") }}.tar.bz2
+  url: http://sourceforge.net/projects/boost/files/boost/{{ version }}/boost_{{ version.replace(".", "_") }}.tar.bz2
```

Comment:
We could assign `version.replace(".", "_")` to another variable if it would be a bit nicer.